dataservice/util/data_gen/data_generator.py

```diff
@@ -162,14 +164,38 @@ def drop_all(self):
         self.app_context.pop()
 
     def create_and_publish_all(self):
-
         # Create participants
         self._create_participants(self.max_participants)
+        # Create family relationships
+        self._create_family_relationships()
         self.teardown()
 
+    def _pairwise(self, iterable):
+        """
+        Iterate over an iterable in consecutive pairs
+        """
+        a, b = tee(iterable)
+        next(b, None)
+        return zip(a, b)
+
+    def _create_family_relationships(self):
```

Comment:
Might be nice to have more than one family with some participants having multiple first order relations.